src/lib/sql/sql_translator.cpp

```diff
@@ -266,6 +277,26 @@ std::shared_ptr<AbstractLQPNode> SQLTranslator::_translate_select_statement(cons
   return _current_lqp;
 }
 
+void SQLTranslator::_translate_hsql_with_description(hsql::WithDescription& desc) {
+  SQLTranslator with_translator{_use_mvcc, nullptr, _parameter_id_allocator, _with_descriptions};
+  const auto lqp = with_translator._translate_select_statement(*desc.select);
+
+  // Save mappings: ColumnID -> ColumnName
+  std::unordered_map<ColumnID, std::string> column_names;
+  for (auto column_id = ColumnID{0}; column_id < lqp->column_expressions().size(); ++column_id) {
+    const auto identifiers =
+        with_translator._sql_identifier_resolver->get_expression_identifiers(lqp->column_expressions()[column_id]);
```

Comment:
This empty line seems odd.

Reply:
You might want to use `_inflated_select_list_elements[column_id].identifiers` instead of `get_expression_identifiers(...)` since `get_expression_identifiers(...)` would return `{"a", "a1", "a2"}` for both `ColumnID`s in `SELECT a AS a1, a AS a2`.